src/profile/profile.js

```diff
@@ -3,10 +3,46 @@
 const appendCSS = require('../appendCSS');
 const css = require('./profile.css');
 const template = require('./profile.html').default;
+const axios = require('axios');
+const validator = require('validator');
 
 // appendCSS(css);
 
 module.exports = app => app.component('profile', {
   inject: ['auth'],
-  template: template
+  data: function() {
+    return {
+      company: '',
+      description: '',
+      logo: '',
+      name: '',
+      error: ''
+    }
+  },
+  template: template,
+  mounted: async function() {
+    console.log('this.auth', this.auth)
+    this.company = this.auth.user.companyName;
+    this.description = this.auth.user.description;
+    this.logo = this.auth.user.logo;
+    this.name = this.auth.user.githubUsername;
+  },
+  methods: {
+    async updateSub() {
+      if(!validator.isURL(this.logo)) {
+        this.error = 'Please enter a valid url';
+        return;
+      } else {
+        this.error = '';
+      }
+      const res = await axios.post(`http://localhost:7071/api/updateSub`, 
```

Comment:
`await api.post('/api/updateSubscriber')` should work once we address comments on https://github.com/mongoosejs/backend/pull/8#event-5983013890